ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/deneb/util/ForkChoiceUtilDeneb.java

```diff
@@ -54,7 +54,7 @@ private UInt64 computeFirstAvailabilityWindowSlot(final ReadOnlyStore store, fin
     final SpecConfigDeneb specConfigDeneb = SpecConfigDeneb.required(specConfig);
     final UInt64 maybeAvailabilityWindowStartSlot =
         spec.computeStartSlotAtEpoch(
-            currentEpoch.minusMinZero(specConfigDeneb.getMinEpochsForBlobSidecarsRequests()));
+            currentEpoch.minusMinZero(specConfigDeneb.getEpochsStoreBlobs()));
```

Comment:
Hmm think `ForkChoice` should use the spec value `getMinEpochsForBlobSidecarsRequests`